test/test_euler.py

```diff
@@ -371,7 +371,7 @@ def inf_norm(data):
         # (Explanation courtesy of Mike Campbell,
         # https://github.com/illinois-ceesd/mirgecom/pull/44#discussion_r463304292)
 
-        momerr = inf_norm(iff_split.momentum) - p0
+        momerr = abs(inf_norm(iff_split.momentum) - p0)
```

Comment:
`p0` is just a `DOFArray` right? How about we just write `inf_norm(iff_split.momentum - p0)`?

Reply:
Yeah I thought about this.  The thing is that the momentum component here is a ndim x ndim array with only the diagonal being p0, so the off-diag term'll fail the test.